packages/tools/test/npm.coffee

```diff
@@ -0,0 +1,79 @@
+
+nikita = require '@nikitajs/tools'
+{tags, ssh} = require './test'
+they = require('ssh2-they').configure ssh...
+
+return unless tags.tools_npm
+
+describe 'tools.npm', ->
+
+  describe 'schema', ->
+
+    they 'name is required', ({ssh}) ->
+      nikita
+        ssh: ssh
+      .tools.npm
+        global: true
+      , (err, {status}) ->
+        status.should.be.false() unless err
+      .promise()
+
+    they 'is valid', ({ssh}) ->
+      nikita
+        ssh: ssh
+      .tools.npm
+        name: 'coffeescripts'
+      , ({options}) ->
+        options.name.should.be.an.Array()
+        options.global.should.be.Boolean()
+        options.upgrade.should.be.Boolean()
+      .promise()
+
+    they 'use defaults', ({ssh}) ->
+      nikita
+        ssh: ssh
+      .tools.npm
+        name: 'coffeescripts'
+      , ({options}) ->
+        options.global.should.be.false()
+        options.upgrade.should.be.false()
+      .promise()
```

Comment:
This seems like a duplicated of the previous test, either merge them or, better to me, test only the property name in the previous one and the default values in this one like you are doing now.